<html>
<head>
<title>TestNG:  validOrderId</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>validOrderId</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>0/1/0</td>
</tr><tr>
<td>Started on:</td><td>Thu Nov 21 12:26:04 IST 2019</td>
</tr>
<tr><td>Total time:</td><td>31 seconds (31029 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='in.shark.online.web.test.outlet.TestGetOrderByIdApiTestNg.testGetOrdersById()'><b>testGetOrdersById</b><br>Test class: in.shark.online.web.test.outlet.TestGetOrderByIdApiTestNg<br>Parameters: 1234567, app_cp=e7ab992f-b94e-4017-95bd-a5dabb917698, /api/order/id, CH4374700000002</td>
<td><div><pre>java.lang.AssertionError: Status expected:&lt;200 OK&gt; but was:&lt;404 NOT_FOUND&gt;

&gt; GET https://qa3-api.chaipoint.com/outlet-pos-api/api/order/id?orderId=CH4374700000002
&gt; WebTestClient-Request-Id: [1]
&gt; Cookie: [app_cp=e7ab992f-b94e-4017-95bd-a5dabb917698]
&gt; service.auth.key: [1234567]
&gt; content-type: [application/json;charset=UTF-8]

No content

&lt; 404 NOT_FOUND Not Found
&lt; Date: [Thu, 21 Nov 2019 06:56:33 GMT]
&lt; Content-Type: [application/json;charset=UTF-8]
&lt; Transfer-Encoding: [chunked]
&lt; Connection: [keep-alive]
&lt; Server: [Apache-Coyote/1.1]
&lt; X-Content-Type-Options: [nosniff]
&lt; X-XSS-Protection: [1; mode=block]
&lt; Cache-Control: [no-cache, no-store, max-age=0, must-revalidate]
&lt; Pragma: [no-cache]
&lt; Expires: [0]
&lt; X-Frame-Options: [DENY]
&lt; Access-Control-Allow-Headers: [Content-Type, Cookie, service.auth.key]
&lt; Access-Control-Allow-Credentials: [true]
&lt; Access-Control-Max-Age: [86400]

{&quot;timestamp&quot;:1574319393639,&quot;status&quot;:404,&quot;error&quot;:&quot;Not Found&quot;,&quot;message&quot;:&quot;No handler found for GET /outlet-pos-api/api/order/id&quot;,&quot;path&quot;:&quot;/outlet-pos-api/api/order/id&quot;}

	at org.springframework.test.web.reactive.server.ExchangeResult.assertWithDiagnostics(ExchangeResult.java:200)
	at org.springframework.test.web.reactive.server.StatusAssertions.assertStatusAndReturn(StatusAssertions.java:227)
	at org.springframework.test.web.reactive.server.StatusAssertions.isOk(StatusAssertions.java:67)
	at in.shark.online.web.test.outlet.TestGetOrderByIdApiTestNg.testGetOrdersById(TestGetOrderByIdApiTestNg.java:51)
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:180)
Caused by: java.lang.AssertionError: Status expected:&lt;200 OK&gt; but was:&lt;404 NOT_FOUND&gt;
	at org.springframework.test.util.AssertionErrors.fail(AssertionErrors.java:55)
	at org.springframework.test.util.AssertionErrors.assertEquals(AssertionErrors.java:82)
	at org.springframework.test.web.reactive.server.StatusAssertions.lambda$assertStatusAndReturn$4(StatusAssertions.java:227)
	at org.springframework.test.web.reactive.server.ExchangeResult.assertWithDiagnostics(ExchangeResult.java:197)
	... 34 more
... Removed 30 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1111173986", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1111173986'><pre>java.lang.AssertionError: Status expected:&lt;200 OK&gt; but was:&lt;404 NOT_FOUND&gt;

&gt; GET https://qa3-api.chaipoint.com/outlet-pos-api/api/order/id?orderId=CH4374700000002
&gt; WebTestClient-Request-Id: [1]
&gt; Cookie: [app_cp=e7ab992f-b94e-4017-95bd-a5dabb917698]
&gt; service.auth.key: [1234567]
&gt; content-type: [application/json;charset=UTF-8]

No content

&lt; 404 NOT_FOUND Not Found
&lt; Date: [Thu, 21 Nov 2019 06:56:33 GMT]
&lt; Content-Type: [application/json;charset=UTF-8]
&lt; Transfer-Encoding: [chunked]
&lt; Connection: [keep-alive]
&lt; Server: [Apache-Coyote/1.1]
&lt; X-Content-Type-Options: [nosniff]
&lt; X-XSS-Protection: [1; mode=block]
&lt; Cache-Control: [no-cache, no-store, max-age=0, must-revalidate]
&lt; Pragma: [no-cache]
&lt; Expires: [0]
&lt; X-Frame-Options: [DENY]
&lt; Access-Control-Allow-Headers: [Content-Type, Cookie, service.auth.key]
&lt; Access-Control-Allow-Credentials: [true]
&lt; Access-Control-Max-Age: [86400]

{&quot;timestamp&quot;:1574319393639,&quot;status&quot;:404,&quot;error&quot;:&quot;Not Found&quot;,&quot;message&quot;:&quot;No handler found for GET /outlet-pos-api/api/order/id&quot;,&quot;path&quot;:&quot;/outlet-pos-api/api/order/id&quot;}

	at org.springframework.test.web.reactive.server.ExchangeResult.assertWithDiagnostics(ExchangeResult.java:200)
	at org.springframework.test.web.reactive.server.StatusAssertions.assertStatusAndReturn(StatusAssertions.java:227)
	at org.springframework.test.web.reactive.server.StatusAssertions.isOk(StatusAssertions.java:67)
	at in.shark.online.web.test.outlet.TestGetOrderByIdApiTestNg.testGetOrdersById(TestGetOrderByIdApiTestNg.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.MethodInvocationHelper$1.runTestMethod(MethodInvocationHelper.java:182)
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeHookable(MethodInvocationHelper.java:194)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:707)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.AssertionError: Status expected:&lt;200 OK&gt; but was:&lt;404 NOT_FOUND&gt;
	at org.springframework.test.util.AssertionErrors.fail(AssertionErrors.java:55)
	at org.springframework.test.util.AssertionErrors.assertEquals(AssertionErrors.java:82)
	at org.springframework.test.web.reactive.server.StatusAssertions.lambda$assertStatusAndReturn$4(StatusAssertions.java:227)
	at org.springframework.test.web.reactive.server.ExchangeResult.assertWithDiagnostics(ExchangeResult.java:197)
	... 34 more
</pre></div></td>
<td>6</td>
<td>in.shark.online.web.test.outlet.TestGetOrderByIdApiTestNg@6200f9cb</td></tr>
</table><p>
</body>
</html>